src/server/serverSetup.js

```diff
@@ -207,6 +279,21 @@ fluid.defaults("sjrk.storyTelling.server.app.storyTellingHandlers", {
             route: "/clientConfig",
             method: "get"
         },
+        signupHandler: {
```

Comment:
This pull request does not contain the tests and documentation for these new endpoints. Is it planned to be addressed in a separate ticket?

Reply:
The authentication related handlers are included in [testSeverWithStorage.js](https://github.com/jobara/sjrk-story-telling/blob/SJRK-405/tests/server/testServerWithStorage.js) as they are required for saving/editing stories.

Reply:
They are also tested in [/testServerAuthentication.js](https://github.com/jobara/sjrk-story-telling/blob/SJRK-405/tests/server/testServerAuthentication.js)